engines/instance_verification/spec/requests/api/connect/v3/systems/products_controller_spec.rb

```diff
@@ -108,6 +128,98 @@
           expect(response.body).to eq(serialized_service_sap_json)
         end
       end
+
+      context 'when system is connected to SCC' do
```

Comment:
This also needs to move to the `scc_proxy` spec I think, because all the code it tests is from there.